Intersect.Client.Framework/Gwen/Control/TreeNode.cs

```diff
@@ -214,31 +214,38 @@ public IEnumerable<TreeNode> SelectedChildren
         public event GwenEventHandler<EventArgs> Collapsed;
 
         /// <summary>
-        ///     Renders the control using specified skin.
+        /// Renders the control using the specified skin.
         /// </summary>
-        /// <param name="skin">Skin to use.</param>
+        /// <param name="skin">The skin to use.</param>
```

Comment:
:+1: IIRC most of the MSFT docs for .NET use "The" to start descriptions for params/returns, so this is good